### DISCUSSION TITLE
文本检测模型微调后，检测效果较差

### DISCUSSION THREAD

**Discussion by xsbrsg** [PaddlePaddle/PaddleOCR#12280] [Q&A]
参考：https://github.com/PaddlePaddle/PaddleOCR/blob/main/doc/doc_ch/finetune.md 进行模型微调
使用pp_ocrV3 student模型进行模型微调，训练500epoch，检测效果较差，出现检测框不连续、误检、漏检的现象，请问可能是什么原因。模型输出效果如下，后者为预训练模型输出（较好）
![254 (2)](https://github.com/PaddlePaddle/PaddleOCR/assets/61486634/39bccfae-b018-4130-95ed-b2894d4a3a5a)
![254](https://github.com/PaddlePaddle/PaddleOCR/assets/61486634/dfbcf994-aa68-4ee1-859b-1ec42210012b)
![img_10 (2)](https://github.com/PaddlePaddle/PaddleOCR/assets/61486634/581dd78b-953f-423f-ab95-e5964c2b2ebe)
![img_10](https://github.com/PaddlePaddle/PaddleOCR/assets/61486634/eb77cb7f-0460-4ce2-94a8-36f32f6249ae)

**Comment by tink2123** [comment]
猜测和训练数据相关，训练数据对文本框的标注是否都为word-level？模型会拟合到训练数据中，如果需要line-level的推理结果，需要同步提供行级别的标注。

**Reply by pcycccccc** [reply]
我也出现了同样的问题，但我不是很懂什么原因造成的。我想询问一下行级训练需要设置什么吗？我是使用PPOCRLabel来进行标注，另外我发现我每张图片其实都是固定有两个标注的行级标签，但是到处crop_image的时候发现有一些图片crop_image保存的信息有缺失。
![image](https://github.com/PaddlePaddle/PaddleOCR/assets/108455796/cdaf95b8-2b0c-4d89-8b9e-dc58cda394ed)
从这张图像上可以看到有些图像只有crop_0，正确导出应该是有crop_0和crop_1，这是为什么呢？

**Comment by renshujiajia** [comment]
> 猜测和训练数据相关，训练数据对文本框的标注是否都为word-level？模型会拟合到训练数据中，如果需要line-level的推理结果，需要同步提供行级别的标注。

我也遇到了类似的问题，是否是数据集过少的原因呢？我目前只有100张训练集，但是不明白为什么训练得到的模型在训练集上测试的效果还那么差（出现漏检、检测不连续）。

**Comment by UserWangZz** [comment]
和数据量大小有关也和模型结构有关，ppocr使用了轻量化的技术，参数量小意味着对反向传播很敏感，容易出现性能的起伏。可以尝试使用更深的backbone，应该会有改善
